### ISSUE TITLE
Testing accuracy is not increasing

### ISSUE THREAD

**Issue by Abhishek12331** [astorfi/3D-convolutional-speaker-recognition#31]
Hi,

I am getting a training accuracy of 95% on voxceleb but testing accuracy is around 10% only. What can be reason of this?
Speakers=1211
Batch size =100
epoch = 50
Data size = 24000

**Comment by Abhishek12331** [comment]
And how to use the batch norm which is there in speech_cnn.
